kubernetes-modules/pom.xml

```diff
@@ -17,7 +17,7 @@
         <module>k8s-admission-controller</module>
         <module>kubernetes-spring</module>
         <module>k8s-java-heap-dump</module>
-        <module>k8s-operator</module>
+<!--        <module>k8s-operator</module>--> <!-- Migrated to heavy profile -->
```

Comment:
I guess we can remove this here altogether.

Reply:
It is better to keep this comment. In the future, we will know where the definition of this module is.